purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/CarouselComponent.kt

```diff
@@ -108,17 +110,19 @@ class PartialCarouselComponent(
     @get:JvmSynthetic
     val visible: Boolean? = null,
     @get:JvmSynthetic
-    @SerialName("initial_slide_index")
-    val initialSlideIndex: Int? = null,
+    @SerialName("initial_page_index")
+    val initialPageIndex: Int? = null,
     @get:JvmSynthetic
-    val alignment: VerticalAlignment? = null,
+    @SerialName("page_alignment")
+    val pageAlignment: VerticalAlignment? = null,
     @get:JvmSynthetic
     val size: Size? = null,
     @get:JvmSynthetic
-    @SerialName("side_page_peek")
-    val sidePagePeek: Int? = null,
+    @SerialName("page_peek")
+    val pagePeek: Int? = null,
```

Comment:
We might want to do the rename as well in the `CarouselComponentStyle` and the `CarouselComponentState`? NABD though, I think it's also fine as it is 🤷‍♂️